CHANGELOG.md

```diff
@@ -17,6 +17,10 @@
   [Norio Nomura](https://github.com/norio-nomura)
   [Syo Ikeda](https://github.com/ikesyo)
 
+* Color literals count as single characters to avoid unintentional line length violations.
```

Comment:
Could you please add 2 trailing spaces to this line for making Markdown adds line ending?